src/me/drton/jmavsim/Simulator.java

```diff
@@ -39,8 +39,8 @@
     }
     private static Port PORT = Port.UDP;
 
-    public static boolean   COMMUNICATE_WITH_QGC  = true;   // open UDP port to QGC
-    public static boolean   COMMUNICATE_WITH_SDK  = true;   // open UDP port to SDK
+    public static boolean   COMMUNICATE_WITH_QGC  = false;   // open UDP port to QGC
```

Comment:
For this we'll need to add the -q flag to `sitl_run.sh`.

Reply:
Oh, for SITL, yes!

Reply:
Hang on, no we don't. For SITL the communication to QGC is done from the PX4 side.

Reply:
Right. But then why was this enabled in the first place?

Reply:
I tried to disable the PX4 mavlink instance and it looks like jMAVSim quickly opens the 14550 port and forwards some messages and then stops forwarding traffic.
With `COMMUNICATE_WITH_QGC = false` it works fine, so I suggest we use this change.